src/components/import/render-utils.tsx

```diff
@@ -35,3 +37,31 @@ export const routeDecoratorIcon = (
       return <GitAltIcon style={{ fontSize: radius }} title={t('devconsole~Edit source code')} />;
   }
 };
+
+export type BreadCrumbsProps = {
```

Comment:
Does this need to be exported?

Reply:
Will remove "export".

Reply:
Ok, this isn't needed anymore since we are pulling the BreadCrums from patterfly